twilio/send_sms.rb

```diff
@@ -0,0 +1,13 @@
+# Download the twilio-ruby library from twilio.com/docs/libraries/ruby
+require 'twilio-ruby'
+
+no more
+client = Twilio::REST::Client.new(account_sid, auth_token)
+
+from = '+19735595553' # Your Twilio number
+to = '+447597870934' # Your mobile phone number
```

Comment:
if this is your real number you probably shouldn't post it either

Reply:
Removed as well thanks for the text :)